tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "compilerOptions": {
-    "baseUrl": "src",
+    "baseUrl": ".",
```

Comment:
Does this section make it so we show the about page as our homepage? Or is this just a general Next.js 11 fix?

Reply:
This is just to be able to create the `@app-public` alias properly.

Reply:
Seems this is just a general Next.js 11 fix.

Resolving conversation. 😉